### ISSUE TITLE
Stuck rebooting trying to recover 

### ISSUE THREAD

**Issue by Svobodan** [raspiblitz/raspiblitz#2326]
After an internet outage for half a day raspiblitz 1.6.3 is trying to recover.

![IMG_20210525_160945~2](https://user-images.githubusercontent.com/77900472/119522613-69968a80-bd7c-11eb-8e3e-34a7150c7be4.jpg)

It rebooted about 2 or 3 times then asked to set new Password A after which rebooted again doing recovery.

Is there a way to fix it without loosing opened channels?

**Comment by nyxnor** [comment]
Looks hardware issue. EXT4 is the file system on the ssd. Either way, try to flash the sd card again and/or use a new sd card as they wear out rapidly.

**Comment by Svobodan** [comment]
Even with a new SD card flashed I am getting the same result. Still keeps rebooting (with the same EXT4-fs error) and "recovering". Does that mean that I likely have an SSD failing?

Somewhere I read suggestions that it might be a faulty power supply but changing for another one did not make a difference. Also I have never seen low voltage errors with my raspberry during the 4 month of running it.

**Comment by nyxnor** [comment]
Possible, but more probable is the enclosure. SSDs are reliable, bad enclosures are common.
Try a recommended enclosure, as it is cheaper than a new disk, then if it still does not work, you will need a new disk.

**Comment by Svobodan** [comment]
Alright, thanks for your help. I hope you are right.

**Comment by Svobodan** [comment]
I have gotten a new ssd enclosure (same as the old one which was not in the recommended shopping list) and it did not help. 

Then I tried updating with the newest release 1.7.0 and that seems to have done the trick. EXT4 error was shown again, but only once. After that the recovery was successful. I also had to delete txindex otherwise bitcoind would not start.

Good news is that I am up and running again, bad news is that I cannot tell what the cause of this was. ¯\_(ツ)_/¯
